crates/rpc-types-trace/src/otterscan.rs

```diff
@@ -20,6 +23,37 @@ pub enum OperationType {
     OpCreate2 = 3,
 }
 
+// Implement Serialize for OperationType
+impl Serialize for OperationType {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: Serializer,
+    {
+        serializer.serialize_u8(*self as u8)
```

Comment:
so this is actually a number and not a string?

Reply:
Yes, I think it is:

<img width="434" alt="image" src="https://github.com/user-attachments/assets/c6087e51-e0b2-4636-87b1-f67b87dbadd7">